crates/theme/src/settings.rs

```diff
@@ -154,6 +265,19 @@ pub fn reset_font_size(cx: &mut AppContext) {
     }
 }
 
+struct SystemAppearance(bool);
```

Comment:
I think storing an `Appearance` here would be clearer than storing a `bool`:

```suggestion
struct SystemAppearance(Appearance);
```